primefaces/src/main/java/org/primefaces/metadata/transformer/AbstractInputMetadataTransformer.java

```diff
@@ -72,7 +72,7 @@ else if (input instanceof InputTextarea) {
         return Integer.MIN_VALUE;
     }
 
-    protected boolean isMaxlenghtSet(UIInput input) {
+    protected boolean isMaxLengthSet(UIInput input) {
```

Comment:
It was by design, see the getter or HTML attr name

Reply:
OK i changed the L to lowercase but the word "length" was misspelled "lenght"